README.md

```diff
@@ -24,7 +29,17 @@ Feature Matrix
 | Color Decision List      |  ✖   |   ✖   |
 | Image Sequence Reference |  ✖   |   ✖   |
 
+Requirements
+------------
+
+* [OpenTimelineIO](https://github.com/AcademySoftwareFoundation/OpenTimelineIO) > 14.1 (latest git version currently recommended)
+* [pyaaf2](https://github.com/markreidvfx/pyaaf2) >= 1.4.0
```

Comment:
Is it necessary to list the requirements? They are already set in the `setup.py`. I find that redundant.

Reply:
I personally don't like having to dig into a project files to find such basic information, but I can remove it if preferred.

Reply:
I like the idea of making it clear that you need these two packages (esp pyaaf2), but let's omit the version info here, since that may change often.

Reply:
good idea, I remove the version numbers.